test/commands/index.test.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
```

Comment:
General note for unit tests: On a clean checkout (after npm i) for this branch, tests fail on Windows, for various reasons:

Some are access permissions (Windows is more restrictive sometimes) - the fs error code to handle is EPERM;
Clean ups (removing directories that are not empty is not authorized - the error code here to handle is ENOTEMPTY);
Differences in how stdout is represented (that's for some tests checking what test-worker should display).

Reply:
There may also be memory leaks, as I see warnings during runs: 
(node:13792) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
(node:13792) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added to [process]. Use emitter.setMaxListeners() to increase limit

Reply:
Log for unit test run (platform is Windows): [test-run.log](https://github.com/adobe/aio-cli-plugin-asset-compute/files/4727105/test-run.log)